common/document_parser/issuance_ref.py

```diff
@@ -0,0 +1,100 @@
+"""
```

Comment:
@eebradford Any idea why this is showing up as a new file wasn't this already apart of the repo?  I think we might be running into "merge conflict" with some changes @vctrstrm made?

Reply:
Correct, this files is now `common/document_parser/lib/ref_list.py`.  This branch might need to be rebased off of dev to ensure there aren't any other significant merge conflicts.

Reply:
Is this holding up the PR?